src/store/flow/flow.ts

```diff
@@ -93,8 +143,8 @@ export const mutations: MutationTree<IFlowsState> = {
     const block: IBlock = findBlockWith(blockId, flow) // @throws ValidationException when block absent
     Vue.set(flow, 'firstBlockId', block.uuid)
   },
-  flow_setName(state, { flowId, value }) {
-    findFlowWith(flowId, state as unknown as IContext).name = value
+  flow_setNameFromLabel(state, {flowId, value}) {
+    findFlowWith(flowId, state as unknown as IContext).name = value.replace(/\W+/g, '')
```

Comment:
I'll switch to label. For the replace, this is taken from what we have for block names (we said we would follow the rules for this in the ticket)

> Update the src/components/interaction-designer/block-editors/LabelEditor.vue component to update the flow name behind the scenes when the label is edited to a string conforming to the same rules as block-editors/NameEditor.

Reply:
Ohh, I see! From my perspective, it would be really nice to see "Hello quick brown fox" result in "hello_quick_brown_fox", I don't know what ticket that would be a part of, though. If it doesn't feel like it should be a part fo this changeset, that's also sounds totally fine to me.